test/unit/mdc-drawer/mdc-drawer.test.js

```diff
@@ -44,22 +50,28 @@ function getFixture() {
         </nav>
       </div>
     </div>
+    ${scrimEl}
   </div>
-`;
+  `;
 }
 
-function setupTest() {
-  const root = getFixture();
+function setupTest(variantClass = cssClasses.DISMISSIBLE) {
+  const root = getFixture(variantClass);
   const drawer = root.querySelector('.mdc-drawer');
   const component = new MDCDrawer(drawer);
   const MockFoundationCtor = td.constructor(MDCDismissibleDrawerFoundation);
   const mockFoundation = new MockFoundationCtor();
   return {root, drawer, component, mockFoundation};
 }
+
+function hasClassMatcher(className) {
+  return td.matchers.argThat((el) => el.classList && el.classList.contains(className));
+}
+
 suite('MDCDrawer');
 
 test('attachTo initializes and returns a MDCDrawer instance', () => {
-  const drawer = getFixture().querySelector('.mdc-drawer');
+  const drawer = getFixture(cssClasses.DISMISSIBLE).querySelector('.mdc-drawer');
```

Comment:
Out of curiosity, what happens right now if someone mistakenly instantiates Drawer JS for a standard drawer? I'm wondering if we can do anything to clearly indicate user error in that case.

Reply:
Yah, it can throw an Error. Added an extra else statement to component.

Also added a test case for this.